### PR TITLE
50 add a vignette to tutorial the ar grid mockup generation

### PR DESCRIPTION
**Pull request by LittleBeannie**
Add  `ar-grid.Rmd` to show the generation of AR grid by `metalite`

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -31,7 +31,8 @@ Suggests:
     r2rtf,
     rmarkdown,
     stringr,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    gt
```
Comment (nanxstats) [inline DESCRIPTION:35]: I think this was listed in alphabetical orders

File: vignettes/ar-grid.Rmd [hunk 2]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Create A&R Grid"
+output:
+  rmarkdown::html_vignette
+resource_files:
+   - package_manual.pdf
```
Comment (nanxstats) [inline vignettes/ar-grid.Rmd:6]: Do you really need this?

File: vignettes/ar-grid.Rmd [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+library(metalite)
+library(r2rtf)
+library(dplyr)
+```
+
+## Overview
+
+In this document, we illustrate how to create the A&R grid by `metalite`.
+
+## Build a Meta Data
```
Comment (nanxstats) [inline vignettes/ar-grid.Rmd:30]: "data" is a singular uncountable noun, so use either "Build a metadata object" or "Build metadata".

Also, avoid using title case for headings: https://practicaltypography.com/headings.html

File: vignettes/ar-grid.Rmd [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+### Step 1: input the population and observation datasets.
```
Comment (nanxstats) [inline vignettes/ar-grid.Rmd:32]: 1. input → digest?
2. Avoid the ending `.` in headings
3. Need a blank line between headings and code block below

File: vignettes/ar-grid.Rmd [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+```{r}
+my_plan <- plan(analysis = "ae_summary", population = "apat", 
+             observation = c("wk12", "wk24"), parameter = "any;rel;ser") |>
+  
+  add_plan(analysis = "ae_specific", population = "apat", 
+           observation = c("wk12", "wk24"), 
+           parameter = c("any", "aeosi", "rel", "ser"))
+```
+
+### Step 3: feed in the analysis plan to the existing meta data.
```
Comment (nanxstats) [inline vignettes/ar-grid.Rmd:50]: Is it `meta data` or `metadata`? Need to spell consistently throughout the package.

**Review by nanxstats — CHANGES_REQUESTED** [review]

File: vignettes/ar-grid.Rmd (hunk 5 above)
Comment (LittleBeannie) [reply vignettes/ar-grid.Rmd:50]: Metadata is used in this package. So I use it throughout the package.

File: vignettes/ar-grid.Rmd [hunk 6]
```diff
@@ -47,14 +46,16 @@ my_plan <- plan(analysis = "ae_summary", population = "apat",
...
 
-### Step 3: feed in the analysis plan to the existing meta data.
+### Step 3: feed in the analysis plan to the existing metadata
+
 ```{r}
 my_meta <- my_meta |>  define_plan(my_plan)
 ```
 
-### Step 4: define the key words in the above meta data.
+### Step 4: define the key words in the above meta data
```
Comment (nanxstats) [inline vignettes/ar-grid.Rmd:55]: "metadata" consistency

File: vignettes/ar-grid.Rmd [hunk 7]
```diff
@@ -92,11 +94,13 @@ my_meta <- my_meta |>
 ```
 
 ### Step 5: build the meta data
```
Comment (nanxstats) [inline vignettes/ar-grid.Rmd:96]: "metadata" consistency

**Review by nanxstats — APPROVED** [review]

**Review by BrianLang — APPROVED** [review]
looks good to me

File: vignettes/ar-grid.Rmd (hunk 7 above)
Comment (LittleBeannie) [reply vignettes/ar-grid.Rmd]: solved

**Pull request merged**